github-sync/github-data/users.yaml

```diff
@@ -387,6 +407,11 @@ users:
   - username: thelinuxfoundation
     role: admin
     teams: []
+  - username: trevrosen
+    role: member
```

Comment:
You should be `admin`, and can you add Santiago too?

Reply:
Done! Thanks for the review 😸